DataFormats/EgammaCandidates/src/GsfElectron.cc

```diff
@@ -71,6 +71,29 @@ GsfElectron::GsfElectron
  }
 
 GsfElectron::GsfElectron
+ ( int charge, const ChargeInfo & chargeInfo,
+   const GsfElectronCoreRef & core,
+   const TrackClusterMatching & tcm, const TrackExtrapolations & te,
+   const ClosestCtfTrack & ctfInfo,
```

Comment:
ctfInfo appears unnecessary.
Please remove it from this constructor.
Older constructors kept it to preserve interface backward compatibility, which is not applicable here.